x-pack/legacy/plugins/code/public/monaco/override_monaco_styles.scss

```diff
@@ -1,4 +1,4 @@
-.codeContainer__root {
+.codeContainer__monaco {
```

Comment:
@daveyholler @cchaos I renamed this because it was doing double duty: it applied both the monaco fixes here and also applied some layout rules in a layout file. `.codeContainer__monaco` is now _solely_ for monaco fixes, and `.codeContainer__root` is for the layout stuff.

Additionally, our two uses of monaco, in our CodeBlock and EditorComponent components, both have the `codeContainer__monaco` class applied internally so that consumer's don't need to worry about it.